samples/extensions/ray_queries/ray_queries.h

```diff
@@ -80,12 +74,12 @@ class RayQueries : public ApiVulkanSample
 	std::unique_ptr<vkb::core::Buffer> uniform_buffer{nullptr};
 
 	// Ray tracing structures
-	VkPhysicalDeviceAccelerationStructureFeaturesKHR acceleration_structure_features{};
-	AccelerationStructure                            top_level_acceleration_structure{};
-	AccelerationStructure                            bottom_level_acceleration_structure{};
-	uint64_t                                         get_buffer_device_address(VkBuffer buffer);
-	void                                             create_top_level_acceleration_structure();
-	void                                             create_bottom_level_acceleration_structure();
+	VkPhysicalDeviceAccelerationStructureFeaturesKHR  acceleration_structure_features{};
+	std::unique_ptr<vkb::core::AccelerationStructure> top_level_acceleration_structure    = nullptr;
+	std::unique_ptr<vkb::core::AccelerationStructure> bottom_level_acceleration_structure = nullptr;
```

Comment:
I do not think the assignment to nullptr is needed here. I believe the default constructor of std::unique_ptr is doing it.